src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age > 0 && (
+      <>
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      </>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my `}
+        {person.sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
we just need to decide which text will be rendered inside Person__partner element.
and better to make two separate ternary expressions in component for make code more readable.
```
const partner = person.sex === 'm' ? 'wife' : 'husband';
const personPartner = person.isMarried 
  ? `${person.partnerName} is my ${partner}`
  :  'I am not married';
```

```suggestion
      <p className="Person__partner">
        {personPartner}
      </p>
```